.eslintrc.json

```diff
@@ -0,0 +1,30 @@
+{
+  "env": {
+    "browser": true,
+    "jasmine": true,
+    "jquery": true,
+    "amd": true
+  },
+  "extends": "airbnb-base/legacy",
+  "globals": {
+    "edx": true,
+    "RequireJS": true
+  },
+  "parserOptions": {
+      "ecmaVersion": 5
+  },
+  "rules": {
```

Comment:
I'm wondering if we want "quotes" to define ``"avoidEscape": true`` so that we can switch to double quotes when we want to avoid escaping single quotes inside strings:

http://eslint.org/docs/rules/quotes